File: backend/src/basico/002-ejercicios/004-valor_ascii/ascii.js
'use strict'

if (require.main == module){
    var caracter = "a";
    var caracter_en_ascii = caracter.charCodeAt(0);
    console.log("El caracter '" + caracter + "' en ASCII ---> " + caracter_en_ascii);

    var numero = 87;
    var numero_en_caracter = String.fromCharCode(numero);
    console.log("El número " + numero + " en caracter ---> " + numero_en_caracter);
}

File: backend/src/basico/002-ejercicios/004-valor_ascii/ascii.go
package main

import (
    "fmt"
)

func main() {
    // Caracter (es realmente un número)
    caracter := 'a'
    fmt.Printf("Caracter: '%c'\n", caracter)

    // Convertir de caracter a ASCII
    caracter_en_ascii := int(caracter)
    fmt.Printf("ASCII: %d\n\n", caracter_en_ascii)

    // Caracter
    ascii := 97
    fmt.Printf("ASCII: '%d'\n", ascii)

    // Convertir de ASCII a caracter
    ascii_en_caracter := string(ascii)
    fmt.Printf("Caracter: '%s'\n", ascii_en_caracter)
}
